tests/ConfigurationTest.php

```diff
@@ -23,17 +23,6 @@ public function testDoesNotAcceptBadApiKey()
         new Configuration([]);
     }
 
-    public function testDefaultEndpoint()
-    {
-        $this->assertSame($this->config->getNotifyEndpoint(), 'https://notify.bugsnag.com');
```

Comment:
There should probably be an equivalent test to ensure the default guzzle configuration points to the endpoint.